docs/changelog.rst

```diff
@@ -16,6 +16,8 @@ Updates
 * CB ThreatHunter
     * Fix List object that was not callable.
 
+.. top-of-changelog (DO NOT REMOVE THIS COMMENT)
```

Comment:
This appears to have fallen below the past few changes. Does this need moved to the top?

Reply:
Yes, it does. Good catch!